docs/docs/reference/terraform.md

```diff
@@ -0,0 +1,22 @@
+# Terraform Usage
```

Comment:
> the new page does not show up in the sidebar

You need to add this page to sidebars.js. If you then set the version to Next, it will show up.